ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/template-types/.content.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:rep="internal"
+    jcr:mixinTypes="[rep:AccessControllable]"
     jcr:primaryType="cq:Page">
     <empty-experience-fragment/>
-    <empty-page/>
     <rep:policy/>
+    <hclecomm-blank-page/>
```

Comment:
@DeepaliAgrwl  : why do we have this here ?

Reply:
@rRishabh this file contents the list of all the templates inside this folder. It is automatically updated when we create or delete any template inside this folder .